components/certs.py

```
@@ -0,0 +1,147 @@
+        ]
+        for path in linux_paths:
+            if os.path.isfile(path):
+                return path
+
+    # --- For macOS ---
+    elif system == "Darwin":
+        # Try system keychain first
+        system_bundle = pathlib.Path("macos-native-ca-bundle.pem").resolve().absolute().as_posix()
+        sh(
+            "security find-certificate -a -p "
+            "/System/Library/Keychains/SystemRootCertificates.keychain "
```

Ruff (0.11.9)

Implicitly concatenated string literals over multiple lines

(ISC002)